test/integration.sh

```diff
@@ -38,7 +38,7 @@ sleep 10
 if ./up \
   --listen=0.0.0.0:8888 \
   --endpoint-type=metrics \
-  --endpoint-read=http://127.0.0.1:9091/api/v1/query \
```

Comment:
This is the breaking change I had in mind - I don't know if we have other users of `up` besides us, but if they would continue to use the full API path, their queries would not work. IMHO it should be fine, just let's keep this in mind :+1:.

Reply:
👍